src/node/coinstats.cpp

```diff
@@ -12,7 +12,8 @@
 #include <serialize.h>
 #include <validation.h>
 #include <uint256.h>
-#include <util/system.h>
+// #include <util/system.h>
```

Comment:
Why did you leave it as a comment instead of just change the line(s) here and the two below?

Reply:
bitcoin 16728 is from 0.19 and these lines are basically hints about how it should look like _once we catch up_, should help resolving future merge conflicts/potential issues.